mongo/mongo.go

```diff
@@ -0,0 +1,362 @@
+//go:build !nomongo
+// +build !nomongo
+
+package mongo
+
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/smallstep/nosql/database"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/writeconcern"
+)
+
+// DB is a wrapper over *sql.DB,
+type DB struct {
+	db *mongo.Database
+}
+
+type Tuple struct {
```

Comment:
`Tuple` likely can be made private; it doesn't need to be imported outside this package.